src/lib/utils/functions/getEndpoint.ts

```diff
@@ -10,7 +10,7 @@ type EndpointObject = {
 }
 
 export const PRODUCTION_ENDPOINT_CONFIG: EndpointObject = {
-  REPO: 'https://repo-prod.prod.sagebase.org/',
+  REPO: 'https://repo-prod.prod.sagebase.org',
```

Comment:
Mock Service Worker is picky about the difference between `https://domain.com/resource` and `https://domain.com//resource`, and we had both the single and double slash styles in our client code. Synapse backend doesn't care about it, so it was never a problem before now.

Getting rid of the slash here makes it harder to accidentally double-slash.

I double-checked that all `'repo'`, `'file'`, and `'auth'` strings in `SynapseClient` are now preceded by a slash, so I am hopeful that this doesn't break anything.